website/content/en/pre-docs/AWS/constraints.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Provisioning Configuration"
+linkTitle: "Provisioning"
+weight: 10
+---
+
+The [Provisioner CRD]({{< ref "../reference/provisioner-crd.md" >}}) provides two sections for constraining nodes. 
+
+- [`spec.requirements`](../reference/provisioner-crd#specrequirements)
+  - Cloud Provider Agnostic 
+  - Kubernetes Well Known Labels
+  - This section includues generally applicable constraints (zone, instance type) that each cloud provider is expected to implement. 
+  - Reference the Providioner CRD for more information. 
+- [`spec.provider`](#specprovider)
+  - Cloud Provider Specific
+  - This section defines constraints that are unique to AWS, such as SecurityGroups.
+  - Defined below. 
+
+## spec.provider
+
+cloud provider specific constrains
+
+[Review these fields in the code.](https://github.com/awslabs/karpenter/blob/main/pkg/cloudprovider/aws/apis/v1alpha1/provider.go#L33)
+
+### InstanceProfile
+An `InstanceProfile` is a region specific EC2 resource that is comprised of a reference to a single global IAM role.
+
+It is required, and specified by name. A suitable `KarpenterNodeRole` is created in the getting started guide.
+
+```
+spec:
+  provider:
+    instanceProfile: MyInstanceProfile
+```
+
+### LaunchTemplate
+
+A launch template is a set of configuration values sufficent for launching an EC2 instance (e.g., AMI, storage spec).
+
+A custom launch templay *may optionally* be specified by name. If none is specified, Karpenter will automatically create a launch template.

```

Comment:
I think "templay" should be changed to "template". Not sure you need the word "optionally" but that's a stylistic thing.